packages/admin/src/FilamentManager.php

```diff
@@ -144,7 +144,7 @@ public function registerStyles(array $styles): void
         $this->styles = array_merge($this->styles, $styles);
     }
 
-    public function registerTheme(string | Htmlable | null $theme): void
```

Comment:
Please remove all formatting changes from this PR

Reply:
done?

Reply:
done?